theia-extensions/viewer-prototype/src/browser/trace-explorer/trace-explorer-sub-widgets/trace-explorer-keyboard-shortcuts/events-table-shortcuts-table.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+
+export class EventsTableShortcutsTable extends React.Component {
+
+    render(): React.ReactNode {
+        return <React.Fragment>
+            <span className='shortcuts-table-header'>EVENTS TABLE NAVIGATION</span>
```

Comment:
Should this be more generically "TABLE NAVIGATION" ?

It applies to output type "TABLE" which covers the Events Table but also now some instances of Latency Table.